scalafix-core/shared/src/main/scala/scalafix/internal/rule/RemoveUnusedImports.scala

```diff
@@ -7,7 +7,10 @@ import scalafix.rule.RuleCtx
 import scalafix.rule.SemanticRule
 
 case class RemoveUnusedImports(index: SemanticdbIndex)
-    extends SemanticRule(index, "RemoveUnusedImports") {
+    extends SemanticRule(
+      index,
+      "RemoveUnusedImports",
+      "Semantic rule that remove unused import declarations.") {
```

Comment:
Rewrite that removes unused imports reported by the compiler under -Xwarn-unused-import.